app/src/main/kotlin/com/simplemobiletools/smsmessenger/activities/ThreadActivity.kt

```diff
@@ -847,6 +847,7 @@ class ThreadActivity : SimpleActivity() {
     private fun sendMessage() {
         var msg = thread_type_message.value
         if (msg.isEmpty() && attachmentSelections.isEmpty()) {
+            toast(R.string.unknown_error_occurred, length = Toast.LENGTH_LONG)
```

Comment:
lets just use showErrorToast instead, it will be cleaner as all these toasts are errors anyway

Reply:
[done!]([e73117b](https://github.com/SimpleMobileTools/Simple-SMS-Messenger/pull/388/commits/e73117b872e695610c080b973f15739e7fc28247))